### ISSUE TITLE
Do not enable Style/Exponential Notation by default

### ISSUE THREAD

**Issue by stomar** [rubocop/rubocop#12653]
For numbers in scientific notation, at least two different styles are widely used, normalized scientific notation and engineering notation. Enforcing one of those styles by default is IMO too intrusive.

~~~ ruby
lambda1 = 400e-9
distance = 12.5e-2
~~~

rubocop 1.60.2:

~~~
Style/ExponentialNotation: Use a mantissa in [1, 10[.
~~~

By default, the cop is enabled and enforces **normalized** scientific notation.
But depending on the field, **engineering** notation might be more common.
It might even make sense to use neither of those two (e-2 for distances in cm).

Enabling it by default makes it necessary to add yet another configuration setting.

--------

## Expected behavior

RuboCop should not report this by default, Style/Exponential Notation should be disabled.

## Actual behavior

Style/Exponential Notation is enabled by default.

## Steps to reproduce the problem

See example above.

## RuboCop version

```
$ [bundle exec] rubocop -V
1.60.2 (using Parser 3.3.0.5, rubocop-ast 1.30.0, running on ruby 3.3.0) [x86_64-linux]
```

**Comment by koic** [comment]
Personally inclined to respect writer's intentions, this proposal seems acceptable.

@tdeo You have proposed the Style Guide rule https://github.com/rubocop/ruby-style-guide/pull/811 and its implementation https://github.com/rubocop/rubocop/pull/7851 for this issue. Do you have any thoughts on this matter?

**Comment by tdeo** [comment]
While I don't have a strong personal attachment to the rule, I do believe consistency is king when it comes to using that notation to help the reader.

Given the very low amount of times this has been discussed over nearly 4 years, I do believe the current configuration is reasonable enough even though it won't please 100% of people

**Comment by dvandersluis** [comment]
This has had no further comments in over a year so I'm of the mind that the existing configuration is fine.
